src/tools/fuzzing.h

```diff
@@ -2696,34 +2743,48 @@ class TranslateToFuzzReader {
         .add(FeatureSet::ReferenceTypes | FeatureSet::ExceptionHandling,
              Type::exnref));
   }
-  Type getReachableType() { return pick(getReachableTypes()); }
 
-  std::vector<Type> getConcreteTypes() {
-    return items(
-      FeatureOptions<Type>()
-        .add(FeatureSet::MVP, Type::i32, Type::i64, Type::f32, Type::f64)
-        .add(FeatureSet::SIMD, Type::v128)
-        .add(FeatureSet::ReferenceTypes,
-             Type::funcref,
-             Type::anyref,
-             Type::nullref)
-        .add(FeatureSet::ReferenceTypes | FeatureSet::ExceptionHandling,
-             Type::exnref));
+  Type getSingleConcreteType() { return pick(getSingleConcreteTypes()); }
+
+  Type getTupleType() {
+    std::vector<Type> elements;
+    size_t numElements = 2 + upTo(MAX_TUPLE_SIZE - 1);
```

Comment:
why 2 and not 1? with 1 we'd also test tuples of size 1 (or do we disallow that?)

Reply:
We do disallow tuples of size 1.

Reply:
Hmm, nope. Looks like it's not even checked in the validator.